backend/src/main/resources/db/migration/V1.18__add_compression_dict_table.sql

```diff
@@ -0,0 +1,8 @@
+create table compression_dictionaries
+(
+    id            SERIAL primary key,
```

Comment:
```suggestion
    id            integer GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
```

Apparently this is more modern and portable